data/sandramsc.json

```
@@ -4,22 +4,26 @@
   "tags": [
     "React.js",
     "Python",
-    "Typescript",
-    "WordPress",
     "Open Source",
     "Artificial Intelligence (AI)",
+    "WordPress",
     "Web3"
   ],
   "socials": [
     {
       "icon": "FaLinkedin",
-      "color": "#0000FF",
+      "color": "#000000",
       "url": "https://linkedin.com/in/sandraashipala"
     },
     {
       "icon": "FaGithub",
       "color": "#000000",
       "url": "https://github.com/sandramsc"
+    },
+    {
+      "icon": "FaTwitter",
+      "color": "#000000",
+      "url": "https://twitter.com/skogsglaenta"
     }
   ],
   "links": [
```

[Nitpick/Question] This is most likely the Milestones section. Is that what you meant to put as the name?
```suggestion
  "links": [
```

---

@CBID2 ..the milestone section is displayed below the mentioned section.

---

@CBID2 these JSON properties match links section (not the milestones). Also in view of @sandramsc comment I will mark this as resolved.